pkg/routeagent_driver/handlers/kubeproxyiptables/cleanup_handling.go

```diff
@@ -13,7 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package kubeproxy_iptables
+package kubeproxyiptables
```

Comment:
I think the '_' makes it more readable.  How about we just make it _kubeproxy_?

Reply:
agreed about readability but I don't see a good match that works with the "kubeproxyiptables" directory name? can they be different, if they can be different ... may be

Reply:
directory name and package name can be different but its a convention that they should be the same (not necessarily though) so I changed the directory name.

Reply:
How about kpiptables?

Reply:
@mangelajo Hw about just shorten to _kubeproxy_? But if the iptables part is important, ie there could be another kubeproxy implementation,  then how about a subdirectory _kubeproxy/iptables_?

Re: '_' in general in package names, I'm OK with it although we should still strive for brevity to avoid the need - if others agree, perhaps we can disable this check/failure?

Reply:
Yeah, that makes sense we could have kubeproxy now, and if we ever support the other implementation use folders.